starboard/build/config/BUILD.gn

```diff
@@ -242,7 +243,12 @@ config("default_compiler_flags") {
 # TODO: b/329683364 - Move other flags setting the c++ standard from each
 # platform's platform_configuration to this config.
 config("default_cpp_standard") {
-  if (sb_is_modular && current_toolchain == cobalt_toolchain) {
+  if ((sb_is_modular && current_toolchain == cobalt_toolchain) || is_android) {
```

Comment:
How is tvos covered?

Reply:
i'll test this change on internal builders(includes tvos) and amend this accordingly.

Reply:
I added tvos flags, tested on kokoro, build passes